pkg/util/sys/device.go

```diff
@@ -43,6 +43,14 @@ type Partition struct {
 	Filesystem string
 }
 
+type CephVolumeDeviceData struct {
+	DevicePath string `json:"device path"`
```

Comment:
In the json names I don't think we can have spaces. For example: `devicePath` or `modelName`.

Reply:
They can have spaces. I have used as field names the exact strings used in the "plain" output of ceph-volume inventory.

As example here an extract of the information shown in the nodes configmap:

`"cephVolumeData":{"device path":"/dev/sdb","size":"25.00 GB","rotates":true,"available":true,"model name":"VBOX HARDDISK"}}`